sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/ItemBulkOperation.java

```diff
@@ -3,9 +3,7 @@
 
 package com.azure.cosmos.implementation.batch;
 
-import com.azure.cosmos.BridgeInternal;
-import com.azure.cosmos.CosmosItemOperation;
-import com.azure.cosmos.CosmosItemOperationType;
+import com.azure.cosmos.*;
```

Comment:
code style: don't use wildcard import

change you intellj setting: 
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports